test/pkg/environment/azure/environment.go

```diff
@@ -45,11 +45,11 @@ type Environment struct {
 
 func NewEnvironment(t *testing.T) *Environment {
 	env := common.NewEnvironment(t)
-
-	return &Environment{
-		Region:      "westus2",
+	azureEnv := &Environment{
```

Comment:
What does the introduction of a variable accomplish here?

Reply:
We are assigning a bunch of values to this struct now, so it makes sense once more.